src/encoded/search.py

```diff
@@ -930,10 +978,14 @@ def initialize_facets(request, doc_types, prepared_terms, schemas):
     # hold disabled facets from schema; we also want to remove these from the prepared_terms facets
     disabled_facets = []
 
+    current_type_schema = request.registry[TYPES][doc_types[0]].schema
```

Comment:
Is the first the default? I don't like seeing array references inside array references. Maybe it's a worry one will fail and you won't know which one. But it's also multiple abstractions in play at once. I'd be happier with (untested):
```
default_doc_type = doc_types[0]
current_type_schema = request.registry[TYPES][default_doc_type].schema
```
or something like that.